src/ContinuityAdvElemKernel.C

```diff
@@ -102,12 +102,12 @@ ContinuityAdvElemKernel<AlgTraits>::execute(
   SharedMemView<double**>& v_velocity = scratchViews.get_scratch_view_2D(*velocityRTM_);
   SharedMemView<double**>& v_Gpdx = scratchViews.get_scratch_view_2D(*Gpdx_);
 
-  SharedMemView<double**>& v_scs_areav = scratchViews.scs_areav;
+  SharedMemView<double**>& v_scs_areav = scratchViews.get_me_views().scs_areav;
```

Comment:
This interface looks fine, however, seems more complex than what we need? Let me have more perspective on why this was changed. It may, in the end, help with BCs or FEM?